op-bindings/predeploys/addresses_test.go

```diff
@@ -11,13 +11,17 @@ import (
 	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
 )
 
-// [Kroma: START]
-// func TestGethAddresses(t *testing.T) {
-// 	// We test if the addresses in geth match those in op-bindings, to avoid an import-cycle:
-// 	// we import geth in the monorepo, and do not want to import op-bindings into geth.
-// 	require.Equal(t, L1BlockAddr, types.L1BlockAddr)
-// }
-// [Kroma: END]
+/* [Kroma: START]
+func TestGethAddresses(t *testing.T) {
+	// We test if the addresses in geth match those in op-bindings, to avoid an import-cycle:
+	// we import geth in the monorepo, and do not want to import op-bindings into geth.
+	require.Equal(t, L1BlockAddr, types.L1BlockAddr)
+}
+[Kroma: END] */
```

Comment:
We use this file in kroma-bindings, so it's better to remain op's code here.
```go
func TestGethAddresses(t *testing.T) {
	// We test if the addresses in geth match those in op-bindings, to avoid an import-cycle:
	// we import geth in the monorepo, and do not want to import op-bindings into geth.
	require.Equal(t, L1BlockAddr, types.L1BlockAddr)
}
```